package.json

```diff
@@ -79,6 +79,7 @@
     ],
     "transform": {
       "^.+\\.json5$": "json5-jest",
+      "^.+\\.jsonc$": "json5-jest",
```

Comment:
Here I've taken advantage of the fact that JSON5 is a superset of JSONC, so the same transform satisfies Jest. I initially writing my own JSONC transform, based on [the JSON5 one](https://github.com/RyosukeCla/json5-jest/blob/633f4b94ec8e4a4c9afe243f12cd0e75b564a402/index.js)—it's basically a one-liner—but for some reason I couldn't get Jest to actually run it. 🤷‍♂️